gaffer-as-a-service/gaas-rest/pom.xml

```diff
@@ -251,8 +237,11 @@
             <version>1.12.8</version>
             <scope>test</scope>
         </dependency>
-
-
+        <dependency>
+            <groupId>ch.qos.reload4j</groupId>
+            <artifactId>reload4j</artifactId>
+            <version>1.2.20</version>
+        </dependency>
```

Comment:
Why is the version specified? It should be inherited from the Gaffer parent POM.

Reply:
Removed